### DISCUSSION TITLE
Crystal Plasticity Kernel with backstress

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#28800] [Q&A Modules: Solid mechanics] [answered]
Hello, I noticed the constitutive model used in the crystal plasticity class has no backstress incorporated in it. Is there any of the models to simulate cyclic loading using the CP model that accounts for backstress?

@sapitts @dschwen

**Comment by sapitts** [comment, marked as answer]
There is not a class with backstress included in the current repository

**Reply by Emmanuel-324** [reply]
Is there going to be one in the future?

**Reply by sapitts** [reply]
I do not believe we currently have support to implement such as class. We do take pull requests if you would like to implement such a model :-)
